### PR TITLE
Lps 124338

### PR DESCRIPTION
**Pull request by shuyangzhou**

### CODE REVIEW

**Comment by shuyangzhou** [comment]
ci:forward

File: portal-impl/src/com/liferay/portlet/PortalPreferencesImpl.java [hunk 1]
```diff
@@ -495,30 +443,25 @@ protected String toXML() {
...
+				preferences.entrySet()) {
+
+			PortalPreferenceKey portalPreferenceKey = entry.getKey();
 
 			Element preferenceElement = portletPreferencesElement.addElement(
 				"preference");
 
-			preferenceElement.addElement("name", entry.getKey());
+			preferenceElement.addElement(
+				"name", portalPreferenceKey.toString());
```
Comment (Preston-Crary) [inline portal-impl/src/com/liferay/portlet/PortalPreferencesImpl.java:424]: @shuyangzhou this is where `PortalPreferenceKey.toString()` is used.  Maybe we give it a different method like `getName()`?

**Comment by shuyangzhou** [comment]
ci:stop

**Comment by shuyangzhou** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/9850

**Pull request closed without merging**